Views/Lombiq.Privacy.ConsentBanner.cshtml

```diff
@@ -66,7 +69,7 @@ else
     <script depends-on="jQuery" at="Foot">
         jQuery(($) => {
             $('#@acceptButtonId').click(function () {
-                var antiForgeryToken = $('[name="__RequestVerificationToken"]').val();
+                var antiForgeryToken = '@requestToken';
 
                 $.post('@acceptanceOfConsentURL', { '__RequestVerificationToken': antiForgeryToken })
```

Comment:
I would argue that we don't need the variable `antiForgeryToken` anymore, now that the code is simplified.

Reply:
In fact it would be nicer to skip the JS object altogether and pass the object natively
```
$.post(
    @acceptanceOfConsentURL.JsonHtmlContent(),
    @Json.Serialize(new { __RequestVerificationToken = antiForgeryToken })
```